src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/annotation/packageannotation/Example1.java

```diff
@@ -6,7 +6,8 @@
 </module>
 */
 
+package com.puppycrawl.tools.checkstyle.checks.annotation.packageannotation;
+
 // xdoc section -- start
-@Deprecated
-package com.example.annotations.packageannotation; //violation
+class Example1 {}
```

Comment:
Please put deprecated annotation over package .
You can put `// xdoc section -- start` above it too`

Reply:
hi @romani , looks like, we have already anotate **package com.puppycrawl.tools.checkstyle.checks.annotation.packageannotation** inside package-info.java.

And again we are anotating the same package within **Example1.class** which is usually not possible as In Java, annotations should typically be placed in a special file named **package-info.java**.

And one more thing as we know about @Deprecated anotation is, In Java, the @Deprecated annotation is typically used to mark classes, methods, or fields that are considered outdated or should not be used anymore. It is not meant to be used above package imports in a Java class.

compilations error getting while doing so.
![Screenshot 2023-11-08 213900](https://github.com/checkstyle/checkstyle/assets/76208964/3dfbd266-5b8c-49dd-a682-435ce2283e82)
![Screenshot 2023-11-08 213832](https://github.com/checkstyle/checkstyle/assets/76208964/06ac1e2e-4ab7-40d0-ae99-0bda8a9fec8b)

@romani 
need you input on this

Reply:
@romani any though on this?

Reply:
Try to make move example to another package/folder
Like: com.puppycrawl.tools.checkstyle.checks.annotation.packageannotationexample1